haproxy/state/from_ha.go

```diff
@@ -51,30 +51,44 @@ func FromHAProxy(ha HAProxyRead) (State, error) {
 		if err != nil {
 			return state, err
 		}
-		if len(logTargets) > 1 {
-			return state, fmt.Errorf("expected at most 1 filter for frontend %s, got %d", f.Name, len(filters))
+		if len(logTargets) >= 2 {
+			return state, fmt.Errorf("expected at most 2 filters for frontend %s, got %d", f.Name, len(filters))
```

Comment:
Message is not consistent with condition above.

Reply:
Fixed